build/csv/csv-gen.sh

```diff
@@ -54,6 +70,15 @@ function generate_csv() {
     sed -i "s/name: rook-ceph.v.*/name: rook-ceph-operator.v$CSV_VERSION/g" "$CSV_FILE_NAME"
     sed -i "s/version: 0.0.0/version: $CSV_VERSION/g" "$CSV_FILE_NAME"
 
+    # Update the csi version according to the downstream build env change
+    sed -i "s|quay.io/cephcsi/cephcsi:v3.10.2|$ROOK_CSI_CEPH_IMAGE|g" "$CSV_FILE_NAME"
```

Comment:
Let's use the LATEST_* varible here instead of the image so that it is easier to update images later on.

Reply:
And, all other places below

Reply:
done